torch/csrc/jit/script/init.cpp

```diff
@@ -1330,6 +1330,9 @@ void initJitScriptBindings(PyObject* module) {
         },
         py::arg("proto_string")
       );
+  m.def("_jit_is_traceable_ivalue", [](const py::object& obj) {
```

Comment:
The name of this function is probably too generic—for example, it returns false for some traceable ivalues (tensor, etc.). Possibly someone will use it expecting it to do the right thing and be sad. Can we change either the name or the impl? If we change the impl, can we use it in the tracer so that there is some guarantee it remains correct?